tests/page-objects/default/enketo/pregnancy-visit.wdio.page.js

```diff
@@ -14,6 +14,17 @@ const formDocument = {
     }
   }
 };
+
+const FORM = 'form[data-form-id="pregnancy_home_visit"]';
+const LLIN = 'input[name="/pregnancy_home_visit/safe_pregnancy_practices/malaria/llin_use"]';
+const IRON_FOLATE = 'input[name="/pregnancy_home_visit/safe_pregnancy_practices/iron_folate/iron_folate_daily"]';
+const HIV_TESTED = 'input[name="/pregnancy_home_visit/safe_pregnancy_practices/hiv_status/hiv_tested"]';
+
+const visitOption = (value) => $(FORM +
+  ` input[name="/pregnancy_home_visit/pregnancy_summary/visit_option"][value="${value}"]`);
+const gestationalAgeCorrect = (value) => $(FORM +
+  ` input[name="/pregnancy_home_visit/pregnancy_summary/g_age_correct"][value="${value}"]`);
```

Comment:
100% agree on not mixing concatenation and string interpolation!  I have fixed this.

I do want to push back just a bit on using implicit returns (even for multi-line values).  I personally find the syntax to be adequately readable. Besides just cutting down on redundant syntax, I like how an implicit return makes it very clear that _only one thing_ is happening in this function.

Our [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/#use-arrow-functions) is not super clear on what should be done with multi-line values:

> Omit the return keyword when the entire function definition fits on one line.

So, at a certain level I guess the readability of it is subjective, but it would be best to have clear common standards on how we want our code to look.  With that in mind, I am happy to update this code (and the style guide documentation) if you would prefer that implicit returns are _only used_ when the entire function definition fits on one line.

Also, just to clarify, would you consider this a multi-line value?

```js
const gestationalAgeCorrect = (value) =>
  $(`${FORM} input[name="/pregnancy_home_visit/pregnancy_summary/g_age_correct"][value="${value}"]`);
```

Reply:
> implicit returns are only used when the entire function definition fits on one line.

I think that was the intention of the style guide line. But yeah it isn't clear right now and adding the word "only" would make sense. If you prefer the alternative maybe start a discussion on Slack? 

> Also, just to clarify, would you consider this a multi-line value?

It's a single line value, but a multi line function definition (if that makes sense).

The reason I don't like it is because it's not super obvious if this is returning anything at all, because a one character change to the line above means it returns undefined. If I'm calling this function I want to be able to quickly see what is being returned which in this case means hunting for either a `return` statement or an opening brace on the line above.

The most important info for APIs is what parameters are taken, and what is returned, and I don't think we should obfuscate that to save a line of code.

This is more extreme with the `reports.wdio.page.js` example below.

Reply:
:+1: I have updated the style guide to be explicit about this in the ongoing docs pr: https://github.com/medic/cht-docs/pull/1232

Reply:
I have also gone through and cleaned up the newly added implicit returns in this PR.  cc @tatilepizs, this will be something to keep in find for future code!